content/pages/office-policies.md

```diff
@@ -0,0 +1,50 @@
+Title: Office Policies
+Category: pages
+Status: hidden
+Slug: office policies
+
+
+WiCS OFFICE POLICY (Version 2.0 March 8th 2015) 
+
+PERSONNEL:
```

Comment:
Suggest you use headings here (denoted by `##` formatting) as opposed to capital letters.